cities_light/migrations/0013_auto_20171229_1942.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.8 on 2017-12-29 18:42
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cities_light', '0012_auto_20171229_1839'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='subregion',
+            unique_together=set([('country', 'name'), ('country', 'slug')]),
+        ),
+    ]
```

Comment:
Could you try to redo migrations perhaps with something as such:

rm -rf cities_light/migrations
git reset --soft origin/master
git checkout cities_light/migrations
manage.py makemigrations cities_light -n add_subregion
git add cities_light/migrations
git commit -m "Add SubRegion feature"
git push -f

Reply:
Ok I tried something but I'm not really a git pro so I hope that's not worse now than before...